.travis.yml

```diff
@@ -54,6 +54,7 @@ before_script:
   - cd apps/$APP_NAME
 
 script:
+  - ./occ face:setup
```

Comment:
I guess it really doesn't matter, but isn't it more logical to have `make test` and only after `face:setup`?

Reply:
> I guess it really doesn't matter, but isn't it more logical to have make test and only after face:setup?

I modified it in case there was any test that depended on the models. Change it along with the necessary tests. :wink: